shared/components/SpeakersList/test.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable max-len */
+
+import React from 'react';
+import SpeakersList from '.';
+import { shallow } from 'enzyme';
+
+const speakers = [
+  {
+    id: 'V8cFyioAACsAd0yt',
+    name: 'Christopher Chedeau',
+    company: 'Facebook',
+    twitterHandle: 'Vjeux',
+    githubHandle: 'Vjeux',
+    blogURL: 'http://blog.vjeux.com/',
+    imageURL: '',
+    bio: [
+      {
+        type: 'paragraph',
+        text: 'With two paragraphs',
+      },
+      {
+        type: 'paragraph',
+        text: 'Full description',
+      },
+    ],
+  },
+  {
+    id: 'V6IbaywAACsAN-X7',
+    name: 'Luca Mezzalira',
+    company: 'Massive Interactive',
+    twitterHandle: 'lucamezzalira',
+    githubHandle: 'lucamezzalira',
+    blogURL: null,
+    imageURL: '//res.cloudinary.com/red-badger-assets/image/upload/v1470318218/speaker3_zkq7nl.png',
+    bio: null,
+  },
+];
+
+describe('SpeakersList component', () => {
+  it('can render', () => {
+    shallow(<SpeakersList speakers={speakers} />);
+  });
```

Comment:
Thing is, we need to implement a whole new UI state for the page if there are no speakers. This is missing atm since it was agreed that we already have some speakers 💭